ng-ncc-app/src/app/components/notify-template-preview/notify-template-preview.component.html

```diff
@@ -3,16 +3,26 @@
 
         <!-- Preview of the template. -->
         <app-panel class="govuk-form-group govuk-body" *ngIf="preview">
-            <span *ngIf="_loading">loading...</span>
-            <span *ngIf="!_loading" [innerHTML]="getCustomisedPreview()"></span>
+            <span *ngIf="loading">loading...</span>
+            <span *ngIf="!loading" [innerHTML]="getCustomisedPreview()"></span>
         </app-panel>
 
     </div>
 
     <div class="govuk-grid-column-one-half">
 
+        <!-- A warning for agents not to enter any sensitive data. -->
+        <div class="govuk-warning-text" *ngIf="hasPlaceholders()">
+          <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+          <strong class="govuk-warning-text__text">
```

Comment:
i wonder if this should be `<p><strong>` instead of `<span><strong>`

Reply:
`<span>` is next to `<strong>`, not nested!